docker/dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the rust image as the base image
+FROM rust:latest
+
+# Install the dependencies required by Rust-TEOS
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install libffi-dev libssl-dev pkg-config libleveldb-dev && \
```

Comment:
I do agree here, these are the deps that we needed for `python-teos`, this may need manual testing to see what deps are needed, as in:

Try to deploy the tower without any deps, start adding them one by one when the build fails. We should keep deps as minimal as possible.

Reply:
Ohh yess yess! sure. I will try to reduce deps